lib/EventLogInterceptor.js

```diff
@@ -211,29 +211,34 @@ class EventLogInterceptor {
   }
 
   publishAndAuditLogEvent(url, method, request, response, checkResponseBody, instanceType) {
-    const eventConfig = _.clone(this.getEventConfig(url, method));
-    if (eventConfig === undefined) {
-      return undefined;
-    }
-    if (checkResponseBody) {
-      eventConfig.check_res_body = true;
+    try {
+      const eventConfig = _.clone(this.getEventConfig(url, method));
+      if (eventConfig === undefined) {
+        return undefined;
+      }
+      if (checkResponseBody) {
+        eventConfig.check_res_body = true;
```

Comment:
Should we use underscore here ?

Reply:
We use camelCase for JS variable names and snake case for config values. [Ideally config values are loaded from YML]

Reply:
👍